grails-app/assets/javascripts/meriplan.js

```diff
@@ -638,6 +650,8 @@ function DetailsViewModel(o, project, budgetHeaders, risks, config) {
     self.threats = new GenericViewModel(o.threats, ['threat', 'intervention']);
     self.consultation = ko.observable(o.consultation);
     self.communityEngagement = ko.observable(o.communityEngagement);
+    self.controlApproach = new GenericViewModel(o.controlApproach, ['data1', 'data2']);
```

Comment:
Despite some of the original sections using this, don't use 'data1' / 'data2' as field names as it makes the data not very useful in other contexts (e.g Download and API JSON is pretty meaningless if all of the fields don't have descriptive names).  See "self.threats" for an example of how to use the GenericViewModel with field names.

Reply:
I've pushed the changes for this